techniques/general/G224.html

```diff
@@ -233,39 +233,39 @@ <h2>Examples</h2>
 
 
 <section id="related">
-      <h2>Related Techniques</h2>
+      <h2>関連テクニック</h2>
       <ul>
       <li><a href="../css/C20">C20: Using relative measurements to set column widths so that lines can average 80 characters or less when the browser is resized</a></li>
       </ul>
     </section>
 <section id="tests">
-      <h2>Tests</h2>
-      <section class="procedure" id="procedure-for-list-example"><h3>Procedure for list example</h3>
+      <h2>検証</h2>
+      <section class="procedure" id="procedure-for-list-example"><h3>リストの事例の手順</h3>
          <ol>
-            <li>Display the web page in a user agent where the page can be zoomed, or the user agent can be resized.</li>
-            <li>Zoom in or resize the browser window so that the viewport is equivallent to 320 CSS pixels wide.</li>
-            <li>The list content fits within the width so that it is only necessary so scroll vertically to read the list.</li>
-            <li>Or, each nested list can be horizontally scrolled into view. Only vertical scrolling is necessary to read the content of list items within each specific nested list level.</li>
+            <li>ウェブページを、ページをズームしたり、ユーザエージェントのサイズを変更したりできるユーザエージェントで表示する。</li>
+            <li>ブラウザのウィンドウをズームイン又はサイズ変更し、ビューポートが 320 CSS ピクセル幅に相当するようにする。</li>
+            <li>リストの内容が幅に収まるように、リストを読むには垂直スクロールだけで済む。</li>
```

Comment:
下記の「コードスニペットの事例の手順」の 3. と同様の構文なので、それに合わせてみました。

```suggestion
            <li>リストの内容は幅に収まるため、リストを読むには垂直スクロールだけで済む。</li>
```